pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -428,6 +434,8 @@ func (k *Kubernetes) InitPod(name string, service kobject.ServiceConfig) *api.Po
 				{
 					Name:  name,
 					Image: service.Image,
+					Stdin: service.Stdin,
+					TTY:   service.Tty,
```

Comment:
as opposed to doing it in multiple places you can do it in one place https://github.com/kubernetes-incubator/kompose/blob/master/pkg/transformer/kubernetes/k8sutils.go#L327